### PR TITLE
[full-ci] Update web to 3.4.0

### PR DESCRIPTION
**Pull request by kulmann**
Update web to 3.4.0

### CODE REVIEW

File: tests/acceptance/expected-failures-webUI-on-OCIS-storage.md [hunk 1]
```diff
@@ -51,8 +51,8 @@ Other free text and markdown formatting can be used elsewhere in the document if
 -   [webUIFilesSearch/search.feature:165](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIFilesSearch/search.feature#L165)
 -   [webUIFilesSearch/search.feature:175](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIFilesSearch/search.feature#L175)
 -   [webUIFilesSearch/search.feature:184](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIFilesSearch/search.feature#L184)
--   [webUISharingInternalUsersShareWithPage/shareWithUsers.feature:118](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingInternalUsersShareWithPage/shareWithUsers.feature#L118)
 -   [webUIRenameFiles/renameFiles.feature:249](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIRenameFiles/renameFiles.feature#L249)
+-   [webUISharingInternalUsersShareWithPage/shareWithUsers.feature:118](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingInternalUsersShareWithPage/shareWithUsers.feature#L118)
```
Comment (phil-davis) [inline tests/acceptance/expected-failures-webUI-on-OCIS-storage.md:55]: This is an odd change - just moving the line.

File: tests/acceptance/expected-failures-webUI-on-OCIS-storage.md [hunk 2]
```diff
@@ -119,13 +121,16 @@ Other free text and markdown formatting can be used elsewhere in the document if
 -   [webUISharingInternalUsersExpire/shareWithUsersExpiringShares.feature:239](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingInternalUsersExpire/shareWithUsersExpiringShares.feature#L239)
 
 ### [share indicator in files are not shown until sharing sidebar is opened](https://github.com/owncloud/web/issues/4167)
+-   [webUISharingInternalUsersSharingIndicator/shareWithUsers.feature:99](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingInternalUsersSharingIndicator/shareWithUsers.feature#L99)
```
Comment (phil-davis) [inline tests/acceptance/expected-failures-webUI-on-OCIS-storage.md:124]: Lines have just been moved.

File: tests/acceptance/expected-failures-webUI-on-OCIS-storage.md [hunk 3]
```diff
@@ -252,7 +256,7 @@ Other free text and markdown formatting can be used elsewhere in the document if
 -   [webUISharingFolderPermissionsGroups/sharePermissionsGroup.feature:64](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingFolderPermissionsGroups/sharePermissionsGroup.feature#L64)
 -   [webUISharingFolderPermissionsGroups/sharePermissionsGroup.feature:66](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingFolderPermissionsGroups/sharePermissionsGroup.feature#L66)
 -   [webUISharingFolderAdvancedPermissionsGroups/shareAdvancePermissionsGroup.feature:64](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingFolderAdvancedPermissionsGroups/shareAdvancePermissionsGroup.feature#L64)
--   [webUISharingFolderAdvancedPermissionsGroups/shareAdvancePermissionsGroup.feature:65](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingFolderAdvancedPermissionsGroups/shareAdvancePermissionsGroup.feature#L65)
+-   [webUISharingFolderAdvancedPermissionsGroups/shareAdvancePermissionsGroup.feature:65](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingFolderAdvancedPermissionsGroups/shareAdvancePermissionsGroup.feature#L65)
```
Comment (phil-davis) [inline tests/acceptance/expected-failures-webUI-on-OCIS-storage.md:259]: Just a space added

**Review by phil-davis — APPROVED** [review]
LGTM - there are a couple of non-essential "moves" in the expected-failures file, but they don't hurt anything.

**Comment by kulmann** [comment]
I only copied the expected failures file from the web repo. There were three unexpected passes with that (see latest commit). I hope that's fine.

**Comment by phil-davis** [comment]
> I only copied the expected failures file from the web repo. There were three unexpected passes with that (see latest commit). I hope that's fine.

Feel free to merge. We will sort out any minor shuffling of lines in expected-failures when we do regular commit-id updates.

**Pull request merged**